cb-sol-cli/cmd/erc721.js

```diff
@@ -36,7 +41,7 @@ const addMinterCmd = new Command("add-minter")
     .action(async function (args) {
         await setupParentArgs(args, args.parent.parent)
         const erc721Instance = new ethers.Contract(args.erc721Address, constants.ContractABIs.Erc721Mintable.abi, args.wallet);
-        const MINTER_ROLE = await erc721Instance.MINTER_ROLE()
+        const MINTER_ROLE = "0x9f2df0fed2c77648de5860a4cc508cd0818c85b8b8a1ab4ceeef8d981c8956a6";
```

Comment:
@chzyer  do you think we can remove the hard coded value? If no, let's create an issue so that we can revisit it again